components/remote-environment-broker/internal/broker/catalog.go

```diff
@@ -34,6 +40,14 @@ func (svc *catalogService) GetCatalog(ctx context.Context, osbCtx osbContext) (*
 	resp := osb.CatalogResponse{}
 	resp.Services = make([]osb.Service, 0)
 	for _, re := range reList {
+		enabled, err := svc.reEnabledChecker.IsRemoteEnvironmentEnabled(osbCtx.BrokerNamespace, string(re.Name))
+		if err != nil {
```

Comment:
to discuss: this error will ends up as a http.StatusBadRequest, is it fine?

Reply:
I'll create an issue for that